test/test_sympy_utils.py

```diff
@@ -216,40 +229,26 @@ def test_unary_ref_range(self, fn):
     # This takes about 4s for all the variants
     @parametrize("fn", BINARY_OPS + COMPARE_OPS)
     def test_binary_ref_range(self, fn):
-        vals = [-sympy.oo, *LESS_CONSTANTS, sympy.oo]
+        # TODO: bring back sympy.oo testing for float unary fns
+        vals = LESS_CONSTANTS
         for a, b in itertools.product(generate_range(vals), repeat=2):
             # don't attempt pow on exponents that are too large (but oo is OK)
             if fn == "pow" and b.upper > 4 and b.upper != sympy.oo:
                 continue
             with self.subTest(a=a, b=b):
-                ref_r = getattr(ValueRangeAnalysis, fn)(a, b)
                 for a0, b0 in itertools.product(LESS_CONSTANTS, repeat=2):
                     if a0 not in a or b0 not in b:
                         continue
                     if not valid_binary(fn, a0, b0):
                         continue
                     with self.subTest(a0=a0, b0=b0):
+                        ref_r = getattr(ValueRangeAnalysis, fn)(a, b)
```

Comment:
These have to be pushed in because if the inputs are not valid, the ValueRangeAnalysis call will also blow up